scripts/bin/update-packages

```diff
@@ -183,12 +186,12 @@ _fetch_and_cache_tags() {
 		echo "build run_all: phony ${__PACKAGES[@]}"
 		echo "default run_all"
 	}
-	
+
 	local LATEST_TAGS="$(\
 		F="$(declare -p TERMUX_SCRIPTDIR GITHUB_TOKEN TERMUX_REPOLOGY_DATA_FILE); $(declare -f __main__ | sed 's/__main__ ()//')" \
```

Comment:
I think the main reason we are getting rate limited is because we aren't slowing down our api requests. See https://docs.github.com/en/rest/using-the-rest-api/rate-limits-for-the-rest-api?apiVersion=2022-11-28#about-secondary-rate-limits, maybe we should actually fix this than using such workarounds